src/layout/Breadcrumb.js

```diff
@@ -62,16 +61,16 @@ export default class Breadcrumb extends React.Component {
         if (passive) {
           fragment = (
             <span key={index}>
-              <span className={styles.crumb}>
+              <span style={{ paddingLeft: '0.5em' }}>
```

Comment:
Above test script is throwing error for CSS file. We can revert this once we are done with testing.